pages/api/[[...slug]].js

```diff
@@ -1,38 +1,38 @@
-import { ApolloServer, gql } from "apollo-server-micro";
-import { carsQuery, carsMutation } from "../../modules/cars.resolvers";
-import { carType, carInputType } from "../../modules/cars.graphql";
-import connectDb from "../../utils/db";
+import { ApolloServer, gql } from 'apollo-server-micro';
+import { carsQuery, carsMutation } from '../../modules/cars.resolvers';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7134248)
```suggestion
import { carsQuery, carsMutation } from "../../modules/cars.resolvers";
```